src/feature_types/feature.js

```diff
@@ -55,4 +50,4 @@ Feature.prototype.internal = function(mode) {
   };
 };
 
-module.exports = Feature;
+export default Feature;
```

Comment:
Soon I want to classify the features too. Going with classes makes sense and if we're going to go with classes, we might as well do import vs require.